bluebutton-data-pipeline-rif-extract/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/extract/s3/task/ManifestEntryDownloadTask.java

```diff
@@ -85,6 +98,24 @@ public ManifestEntryDownloadResult call() throws Exception {
 		}
 	}
 
+	/**
+	 * Returns a Base64 encoded MD5chksum for the input file localTempFile.
+	 * 
+	 * @param localTempFile
+	 * @return
+	 */
+	public String getMD5ChkSum(Path localTempFile) {
```

Comment:
Some suggestions on the method signature:
1. This doesn't need to be `public`, does it? You might want it default-scoped for your tests, but nothing outside of the package needs to see it?
2. Having this named "`getNNNN`" is a bit misleading, as it's doing what could be a lengthy calculation -- rather than being just a fast getter. I'd instead name it "`computeMD5ChkSum`"
3. This can (and probably should be) marked `static`. Doesn't use any instance state that I can see.